### PR TITLE
test(SearchBox): add icon tests

### PR DESCRIPTION
**Pull request by dryhten**
re #587

### CODE REVIEW

File: app/components/ui/molecules/PeoplePicker.test.js [hunk 1]
```diff
@@ -163,5 +163,12 @@ describe('PeoplePicker', () => {
       searchFor('s')
       expect(searchWrapper.find('PersonPod')).toHaveLength(0)
     })
+
+    it('filters the people pods when you click on the search icon', () => {
+      const input = searchWrapper.find('input')
+      input.simulate('change', { target: { value: 'annie' } })
+      searchWrapper.find('[data-test-id="search-icon"]').simulate('click')
+      expect(searchWrapper.find('PersonPod').length).toBeGreaterThan(0)
```
Comment (jsms90) [inline app/components/ui/molecules/PeoplePicker.test.js]: This just checks whether any results are displayed at all, no?

Wouldn't this test pass, even if the results hadn't been filtered or changed at all, so long as there were still person pods in the grid?

Comment (dryhten) [reply app/components/ui/molecules/PeoplePicker.test.js]: Yeah, totally forgot all person pods are displayed by default. I'll change it.

**Comment by dryhten** [comment]
Does clicking on the X icon clear the content of the search box only or does it submit as well? I'm not that familiar with the functionality requirements for it.

**Comment by dryhten** [comment]
Ok, I found the initial issue https://github.com/elifesciences/elife-xpub/issues/540, I'll add a test for it then.

File: app/components/ui/molecules/SearchBox.test.js [hunk 2]
```diff
@@ -77,4 +77,15 @@ describe('SearchBox component tests', () => {
       expect(receivedSuggestions).toEqual(expectedSuggestions)
     },
   )
+
+  it('does nothing when clicking on X icon and nothing is typed', () => {
+    wrapper.find('[data-test-id="cross-icon"]').simulate('click')
+    expect(wrapper.find('input').props().value).toEqual('')
+  })
+
+  it('clears thes input field when x icon is clicked after typing something', () => {
```
Comment (jsms90) [inline app/components/ui/molecules/SearchBox.test.js]: Teeny typo "clears the**s** input field"

**Comment by jsms90** [comment]
@dryhten Any thoughts on the `searchFor` naming I mentioned above?

**Comment by jsms90** [comment]
> not add something ridiculous

Oh @dryhten, is there really any need to be so inflamatory? We currently have two ways of triggering a search, only one of which is named `searchFor`. I simply thought it might help for readability. If you don't think it's important, that's fine. It was only a suggestion :woman_shrugging:

**Pull request merged**